src/components/transactions/TxSummary/QueueActions.tsx

```diff
@@ -8,7 +8,7 @@ const QueueActions = ({ tx }: { tx: TransactionSummary }) => {
   const awaitingExecution = isAwaitingExecution(tx.txStatus)
 
   return (
-    <Box my={-1}>
+    <Box my={-1} mr={2} display={'flex'} flexDirection={'row'} alignItems={'center'} justifyContent={'flex-end'}>
```

Comment:
I automatically reformat with prettier and it does display={'flex'} instead of display="flex". If we can find a way to make it so that prettier reformats it the way you suggest I'm all for this change :)

Reply:
I think there is an [eslint plugin](https://github.com/jsx-eslint/eslint-plugin-react/blob/HEAD/docs/rules/jsx-curly-brace-presence.md) out there to automate this but so far we manually made sure to use one over the other. Imo it doesn't matter as long as it is consistent.